lib/quaff-monkey-patches.rb

```diff
@@ -26,26 +26,53 @@
 
 module Quaff
   class Call
-    def send_request(method, body="", headers={})
+    def send_request(method, options={})
+      body = options[:body] || ""
+      headers = options[:headers] || {}
+      new_tsx = options[:new_tsx].nil? ? true : options[:new_tsx]
```

Comment:
The idiom for this is to have a `DEFAULT_OPTS` map and `merge` it with the `options` parameter.  This would clean up this code quite a bit (I think).